src/Microsoft.ML.LightGBM/LightGbmArguments.cs

```diff
@@ -38,9 +38,12 @@ public interface IBoosterParameter
     }
 
     /// <summary>
-    /// Parameters names comes from LightGBM library.
-    /// See https://github.com/Microsoft/LightGBM/blob/master/docs/Parameters.rst.
+    /// Options for LightGBM trainer.
     /// </summary>
+    /// <remarks>
+    /// LightGBM is an external library that's integrated with ML.NET. For detailed information about the parameters
+    /// please see https://github.com/Microsoft/LightGBM/blob/master/docs/Parameters.rst.
+    /// </remarks>
```

Comment:
Is there a version info anywhere that we could put here so that it's clear what version we have and it's auto-updated when we update LightGBM?

Reply:
I don't think we have an auto-update mechanism for external repo version in docs. Right now it's at 2.2.3 which is the latest. I can't think of a way to force-update the docs here, if LightGBM version is included in text.

It's also easy to look at the LightGBM nuget version that comes with ML.NET nugets. That's the most reliable way to find the version I think.


---
In reply to: [263945106](https://github.com/dotnet/machinelearning/pull/2886#discussion_r263945106) [](ancestors = 263945106)